src/index.js

```diff
@@ -69,10 +69,10 @@ ${unused.join(`\n`)}`);
   }
 
   _getFileDepsMap(compilation) {
-    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => ({
-      ...acc,
-      [usedFilepath]: usedFilepath,
-    }), {});
+    const fileDepsBy = compilation.fileDependencies.reduce((acc, usedFilepath) => {
```

Comment:
`const fileDepsBy = new Set(compilation.fileDependencies)` should theoretically be much faster. I'd be very interested to see the results!

Reply:
Where is an example of this `new Set` method you refer to?